buildwheels.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# This script was copied from github.com/marcelm/dnaio. Many thanks to Marcel!
+# The only adaptation was moving the script to manylinux2014.
+#
+# Build manylinux wheels. Based on the example at
+# <https://github.com/pypa/python-manylinux-demo>
+#
+# It is best to run this in a fresh clone of the repository!
+#
+# Run this within the repository root:
+#   ./buildwheels.sh
+#
+# The wheels will be put into the dist/ subdirectory.
+
+set -xeuo pipefail
+
+manylinux=quay.io/pypa/manylinux2014_x86_64
+
+# For convenience, if this script is called from outside of a docker container,
+# it starts a container and runs itself inside of it.
+if ! grep -q docker /proc/1/cgroup && ! test -d /opt/python; then
+  # We are not inside a container
+  docker pull ${manylinux}
+  exec docker run --rm -v $(pwd):/io ${manylinux} /io/$0
+fi
+
+if ! test -d /io/dist; then
+  mkdir /io/dist
+  chown --reference=/io/setup.py /io/dist
+fi
+
+# Strip binaries (copied from multibuild)
+STRIP_FLAGS=${STRIP_FLAGS:-"-Wl,-strip-all"}
+export CFLAGS="${CFLAGS:-$STRIP_FLAGS}"
+export CXXFLAGS="${CXXFLAGS:-$STRIP_FLAGS}"
+
+for PYBIN in /opt/python/cp3[6789]-*/bin; do
```

Comment:
> cibuildwheel is not intended to run on your development machine. Because it uses system Python from Python.org it will try to install packages globally - not what you expect from a build tool! Instead, isolated CI services like those mentioned above are ideal.

I will stick with your script for a while since I do not intend to automate the publishing yet on CI. (It takes ~30 minutes to an hour for each release. So it is not a big deal to automate). But it is indeed good to be aware that maintained programs to automate this task may exist.

Reply:
Ah, now I remember. Perhaps that prevented me from migrating the last time I tried.